refresh/templates/common/RouteTests.swift

```diff
@@ -58,6 +58,32 @@ class RouteTests: XCTestCase {
 
         waitForExpectations(timeout: 10.0, handler: nil)
     }
+    
+    func testHealthRoute() {
+        
+        let printExpectation = expectation(description: "The /health route will print UP, followed by a timestamp.")
+        
+        URLRequest(forTestWithMethod: "GET", route: "health")?
+            .sendForTestingWithKitura { data, statusCode in
+                if let getResult = String(data: data, encoding: String.Encoding.utf8) {
+                    XCTAssertEqual(statusCode, 200)
+                    XCTAssertTrue(getResult.contains("UP"))
+                    XCTAssertTrue(getResult.contains("2017") || getResult.contains("2018") || getResult.contains("2019"))
```

Comment:
Can we use a more dynamic timestamp check here?  This will be a pain to maintain.

Reply:
Updated to dynamically get the current year and make a check that its the same as returned in the test.